core/src/main/java/in/testpress/database/roommigration/RoomMigration4To5.kt

```diff
@@ -0,0 +1,36 @@
+package `in`.testpress.database.roommigration
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+object RoomMigration4To5 {
+
+    val MIGRATION_4_5: Migration = object : Migration(4, 5) {
+        override fun migrate(database: SupportSQLiteDatabase) {
+            database.execSQL(getCreateProductEntity())
+            database.execSQL(getCreateCourseEntity())
+            database.execSQL(getCreateProductCourseEntity())
+        }
+
+    }
+
+    private fun getCreateProductEntity(): String {
```

Comment:
This method not returning an entity, it returns a SQL query. A better name would be `getProductEntityTableCreationSQL`